examples/MULTIPLE_ENDPOINTS.md

```diff
@@ -0,0 +1,29 @@
+# Hitting multiple endpoints with a traffic profile
+
+## Description
+
+Below is an example which will send requests to two endpoints `127.0.0.1:80` and `127.0.0.2:80`, while alternating over two request headers, which contain different paths and hosts.
+
+## Define a traffic profile
```

Comment:
Unless I'm mistaken you've created this term traffic profile here, and it is not used elsewhere. The way you've structured this makes it look like traffic profile is a broader nighthawk term in a way that I think is misleading.

Could we instead discuss it like:

Place a file called `traffic-profile.yaml` in your current working directory.  This will act as your configuration for the file-based request source.